christfellowshipchurch/web-app-v2
# Fix reset password padding bug

### About
The styling of the password reset screen on mobile web needed to be adjusted

### Test Instructions
Open the password reset screen on mobile view and the text should be centered, as seen on screenshot below)

### Screenshots
Mobile Web 

<img width="531" alt="Screen Shot 2021-08-31 at 7 01 34 PM" src="https://user-images.githubusercontent.com/46769629/131588146-3911e289-0aee-4326-ba3e-dbc4755de226.png">

Web 

<img width="1429" alt="Screen Shot 2021-08-31 at 7 01 19 PM" src="https://user-images.githubusercontent.com/46769629/131588156-93595d69-bd61-4f64-8cd4-7529bcd312f1.png">

### Closes Tickets
[[CFDP-1688]](https://christfellowshipchurch.atlassian.net/browse/CFDP-1688)

### Related Tickets
N/A
